corehq/util/dates.py

```diff
@@ -16,10 +16,6 @@ def get_timestamp(date):
     return time.mktime(date.timetuple())
 
 
-def get_timestamp_millis(date):
```

Comment:
Nice attention to detail noticing this isn't used anymore.

Reply:
That is exactly what I do, too. Maybe Fowler will have better ideas (or we'll find someone has a tool/plugin for dead code).

Reply:
Looking around, maybe https://github.com/jendrikseipp/vulture ? I have not yet tried it.

Reply:
I tried something like that (maybe that one, actually) a while back.  There were too many false positives to trust it, though I see now it has a `--min-confidence 100` option, which may help.